src/AbstractNumber.php

```diff
@@ -200,6 +200,22 @@ public function pow($value, int $scale = null): self
         return $this->init($mod);
     }
 
+    public function powmod($value, $modulus, int $scale = null): self
+    {
+        $exponent = $this->getNumberFromInput($value);
+        $modulus = $this->getNumberFromInput($modulus);
+        $scale = $scale ?? self::INTERNAL_SCALE;
+
+        $exponentWithZeroScale = $exponent->toString(0);
```

Comment:
In my opinion, $exponentWithZeroScale is a good solution for now, since the native bcmath function doesn't allow an exponent with a > 0 scale. In the future, we may consider to implement this method without using the native bcmath function. In this way we can work around this whole zero-scale thing.